aol-htb.js

```diff
@@ -137,6 +153,40 @@ function AolHtb(configs) {
         };
     }
 
+    function __generateOneMobileRequest(xSlot) {
+        var protocol = Browser.getProtocol();
+        var baseUrl = protocol + endpointsUrls.oneMobile.get;
+        var requestId = '_' + System.generateUniqueId();
+
+        var requestParams = {
+            cmd: 'bid',
+            dcn: xSlot.dcn,
+            pos: xSlot.pos
+        };
+
+        if (protocol === 'https:') {
+            requestParams.secure = 1;
+        }
+
+        var url = Network.buildUrl(baseUrl, ['bidRequest?']);
+
+        for (var parameter in requestParams) {
+            if (!requestParams.hasOwnProperty(parameter)) {
+                continue;
+            }
+            url += '&' + parameter + '=' + requestParams[parameter];
```

Comment:
I just want to confirm if you expect the url to be of format `endpoint/bidRequest?&param` or `endpoint/bidRequest?paramA&paramB`

Reply:
Hi.

We definitely would expect the second option. 
I've already updated the code to get ride of this extra "&" sign.

Reply:
Hi, I believe it can be done this way. 
```
var result = inspector.validate({
    type: 'object',
    properties: {
        region: {
            type: 'string',
            exec: function (schema, post) {
                if (!validRegions.hasOwnProperty(post)) {
                    this.report('region must be one of the predefined values: ' + Object.keys(validRegions));
                }
                return post;
            }
        },
        networkId: {
            type: 'string',
            minLength: 1
        },
        xSlots: {
            type: 'object',
            properties: {
                '*': {
                    type: 'object',
                    exec: function(schema, post){
                        if (!((post.hasOwnProperty('dcn') && post.hasOwnProperty('pos')) || post.hasOwnProperty('placementId'))){
                            this.report("xSlots must have either dcn and pos, or placementId");
                        }
                        return post;
                    },
                    properties:{
                        placementId: {
                            optional: true,
                            type: 'string',
                            minLength: 1
                        },
                        sizeId: {
                            optional: true,
                            type: 'string',
                            minLength: 1
                        },
                        pageId: {
                            optional: true,
                            type: 'string',
                            minLength: 1
                        },
                        dcn: {
                            optional: true,
                            type: 'string',
                            minLength: 1
                        },
                        pos: {
                            optional: true,
                            type: 'string',
                            minLength: 1
                        }
                    }
                }
            }
        }
    }
}, configs);
```

Please modify this as you need and if you have any questions please let us know.